src/com/sap/piper/tools/JavaArchiveDescriptor.groovy

```diff
@@ -48,16 +77,21 @@ class JavaArchiveDescriptor extends ToolDescriptor {
         return "$javaCall $javaArchive"
     }
 
+    @Override
+    def verifyToolLocation(script, configuration) { }
+
     @Override
     def getMessage() {
-        def configOptions = "Please, configure $name home. $name home can be set "
+        def configOptions = "Please, configure $name. $name can be set "
         if (environmentKey) configOptions += "using the environment variable '$environmentKey'"
         if (environmentKey && stepConfigurationKey) configOptions += ", or "
-        if (stepConfigurationKey) configOptions += "using the configuration key '$stepConfigurationKey'."
+        if (stepConfigurationKey) configOptions += "using the configuration key '$stepConfigurationKey'"
+        configOptions += ", or it must be located on the current working directory."
         return configOptions
     }
 
     def isOnCurrentWorkingDirectory(script) {
+        executableName = 'mta.jar' // default value
```

Comment:
this is wrong. that's the default value for MTA, but it should not be in a JavaArchiveDescriptor class.